src/main/java/com/google/sps/data/DeliverySlot.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.data;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.Date; 
+
+/** 
+ * Class that represents a delivery timeslot using delivery date, start and end times.
+ */
+public class DeliverySlot {
+  private Date deliveryDay;
```

Comment:
I used int to represent the time in seconds because Directions API returns the duration (to get from origin to destination) in seconds. I will change it to java.util.Date.